sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -3501,10 +3500,14 @@ pi_result piKernelCreate(pi_program Program, const char *KernelName,
   ze_result_t ZeResult = ZE_RESULT_ERROR_INVALID_KERNEL_NAME;
   _pi_program::ModuleIterator ModIt(Program);
   while (!ModIt.Done()) {
-    ZeResult =
-        ZE_CALL_NOCHECK(zeKernelCreate, (*ModIt, &ZeKernelDesc, &ZeKernel));
-    if (ZeResult != ZE_RESULT_ERROR_INVALID_KERNEL_NAME)
-      break;
+    uint32_t KernelNum = 0;
```

Comment:
Please add a source code comment about why this check is needed.

Reply:
Hi, @smaslov-intel 
Comment has already been added.
Thanks very much.

Reply:
OK.